config/webpack/plugins/sku-webpack-plugin/index.js

```diff
@@ -163,28 +162,42 @@ class SkuWebpackPlugin {
       {
         test: IMAGE,
         include: this.include,
-        use: makeImageLoaders({ target }),
+        type: 'asset',
+        generator: {
+          emit: target === 'browser',
+        },
+        parser: {
+          dataUrlCondition: {
+            maxSize: 10000,
+          },
+        },
       },
       {
         test: SVG,
         include: this.include,
+        type: 'asset/source',
         use: makeSvgLoaders(),
       },
-      {
-        test: DEPRECATED_CSS_IN_JS,
-        include: this.include,
-        use: require.resolve('../deprecatedCssInJsFileLoader'),
-      },
     ];
 
     compiler.options.module.rules.push(...rules);
 
-    if (!compiler.options.resolve.extensions.includes('.ts')) {
-      compiler.options.resolve.extensions.push('.ts');
-    }
+    if (!compiler.options.resolve.extensions) {
+      compiler.options.resolve.extensions = [
+        '.mjs',
+        '.js',
+        '.json',
+        '.ts',
+        '.tsx',
+      ];
```

Comment:
I know this is pretty standard, but would it be better to share this list between the webpack config and here to make sure they are in-sync?